library/file/inode.h

```diff
@@ -38,18 +38,20 @@
 #pragma once
 #include "disk.h"
 
+typedef unsigned int uint;
```

Comment:
Why adding `typedef` here? Can we avoid duplicate `typedef` and only do `typedef` in egos.h?

Reply:
From what I have seen, **inode.h** does not have any access to the type definitions in **egos.h**, and when trying to include **egos.h** inside of **inode.h**, conflicts with **printf()** in **file.c** came up. 

What do you believe the best solution would be? We could try using a **types.h** file such as what we were talking about, but I'm not sure what would be best.